birdhouse/components/proxy/docker-compose-extra.yml

```diff
@@ -22,3 +22,10 @@ services:
       REQUESTS_CA_BUNDLE: /etc/ssl/certs/ca-certificates.crt
     restart: always
     logging: *default-logging
+    healthcheck:
+      test: ["CMD", "curl", "--fail", "${BIRDHOUSE_FQDN_PUBLIC}"]
```

Comment:
Ideally we should use an internal endpoint/port to avoid loop-back requests.

Also, consider using another endpoint such as the `/version`, since the root path gets redirected to whichever "frontend" entrypoint is defined, which might be inconsistent or pinging another redirected service from the stack.

Reply:
I like the idea to use `/version` and an internal endpoint. 

Just a note: curl won't follow redirects unless the `--location` flag is used so we don't have to worry about redirects to inconsistent places in this case.